broadcast/src/main/java/org/elasticsoftware/elasticactors/broadcast/Broadcaster.java

```diff
@@ -40,7 +41,7 @@
  * @author Joost van de Wijgerd
  */
 @Actor(stateClass = BroadcasterState.class, serializationFramework = JacksonSerializationFramework.class)
-@PersistenceConfig(persistOnMessages = false, included = {Add.class, Remove.class, UpdateThrottleConfig.class}, persistOn = {CREATE})
```

Comment:
I would consider being backwards compatible with the previous solution

Reply:
UpdateThrottleConfig doesn't do anything anymore. There's no point having it here.

Reply:
ok